azure-steps-windows.yml

```diff
@@ -14,7 +14,7 @@ steps:
     # to $PYTHON_EXE's directory since that is on a different drive which
     # mingw does not like. Instead copy it to a directory and set OPENBLAS,
     # since OPENBLAS will be picked up by the openblas discovery
-    python -m pip  install
+    python -m pip install
```

Comment:
I think this line was left here by mistake: it should be removed.

Reply:
Thanks.  I couldn't imagine what it was doing.